templates/default/less/variables.less

```diff
@@ -345,4 +346,26 @@
 //
 //##
 @il-divider-bg: darken(@il-secondary-container-bg, 5%);
-@il-divider-color: lighten(@text-color, 10%);
\ No newline at end of file
+@il-divider-color: lighten(@text-color, 10%);
+
+//== Item
+//
+//##
+@il-item-divider-padding: 10px 0 0 0;
+@il-item-divider-margin: 0 0 10px 0;
+@il-item-divider-border-bottom: dashed 2px #E0E0E0;
+@il-item-description-padding: 10px 0 0 0;
+@il-item-padding: 20px;
+@il-item-marker-border-left: 5px solid;
+@il-item-h5-margin: 4px 0px 2px 0px;
+
+//== Item
+//
+//##
+@il-panel-listing-std-container-h4-margin: 16px 0 2px 0;
+@il-panel-listing-std-container-h3-margin: 8px 0 2px 0;
+@il-panel-listing-std-container-h3-text-transform: uppercase;
+@il-panel-listing-std-container-h4-text-transform: uppercase;
```

Comment:
This is quite a long name. Note that you can also add a description for a variabel by adding a 
//** Variable 1 Description (see: https://github.com/ILIAS-eLearning/ILIAS/tree/trunk/templates#naming). This is also visible in the GUI in the administration. However, since you chose very systematic names, I think you can leave it as is.